Sources/App/main.swift

```diff
@@ -1,13 +1,20 @@
-import Vapor
+import AppLogic
```

Comment:
Renaming `App` to something like `Run` could be a nice change. That way it's extra obvious what you're supposed to run to boot your server

```
.build/release/Run
```

It's also what `vapor run ...` will run.

Then we can use `App` for the main module instead of `AppLogic`